src/components/map/map-container.vue

```diff
@@ -44,6 +54,7 @@ provide('olMap', olMap)
     <zoom-to-extent-control :extent="DEFAULT_EXTENT" />
     <fullscreen-control />
     <attribution-control />
+    <map-3d-control v-if="v4_standalone" />
```

Comment:
Not a big fan of this, but I understand the need.
Maybe we should handle this with vuejs slots? But lets keep this as is for now.